requirements.txt

```diff
@@ -1,10 +1,10 @@
-livekit-agents>=0.12.17,<1.0.0
-livekit-plugins-openai>=0.11.2,<1.0.0
-livekit-plugins-cartesia>=0.4.10,<1.0.0
-livekit-plugins-deepgram>=0.7.0,<1.0.0
-livekit-plugins-silero>=0.7.5,<1.0.0
+livekit-agents>=1.0.0,<2.0.0
```

Comment:
should just be `~=1.0`, and you should use the [xx] format for the plugins

Reply:
I just updated the versions in requirements.txt, I fo think `agent.py` is a good naming convention for us and think I will keep with it as I migrate other apps.